### DISCUSSION TITLE
Its more efficient to use [[aa]] [[bb]] than [[aa bb]] when add more same level items

### DISCUSSION THREAD

**Discussion by qjcjerry** [SkepticMystic/breadcrumbs#488] [Ideas]
Its more efficient to use [[aa]] [[bb]] than [[aa bb]] when add more same level items，because you can quick search items when use [[]].

**Comment by michaelpporter** [comment]
You can already do this — a typed-link field accepts multiple links, either inline `same:: [[aa]], [[bb]]` or as a YAML list:

```yaml
same:
  - "[[aa]]"
  - "[[bb]]"
```

Each link becomes its own edge, and `[[ ]]` autocomplete works per item as you noted.
